src/platform/packages/shared/kbn-unified-field-list/src/components/field_list_filters/field_type_filter.tsx

```diff
@@ -204,6 +204,14 @@ export function FieldTypeFilter<T extends FieldListItem = DataViewField>({
             data-test-subj={`${testSubj}Options`}
             items={availableFieldTypes.map((type) => (
               <EuiContextMenuItem
+                aria-label={i18n.translate('unifiedFieldList.fieldTypeFilter.typeButtonAriaLabel', {
+                  defaultMessage: '{type} field count: {count}',
+                  values: {
+                    type: getFieldTypeName(type),
+                    count: typeCounts?.get(type) ?? 0,
```

Comment:
Let's introduce variables for data which is used multiple times now like type name, type count, tooltip id.